vyapp/stdout.py

```diff
@@ -38,6 +38,8 @@ def remove(self, fd):
 
     def write(self, data):
         for ind in self.base: 
+            if sys.version_info >= (3, 0):
+                data = str(data)
```

Comment:
Seems like it doesn't matter much either way? (would get removed when Py2x is dropped).